GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -2835,7 +2835,7 @@ private void GoToMergeBase()
                 revisions
             };
 
-            string mergeBaseCommitId = UICommands.Module.GitExecutable.GetOutput(args).TrimEnd('\n');
+            string mergeBaseCommitId = UICommands.Module.GitExecutable.GetOutput(args, throwOnErrorExit: false).TrimEnd('\n');
             if (string.IsNullOrWhiteSpace(mergeBaseCommitId))
```

Comment:
-1
throwOnErrorExit was not added to GetOutput() as the exit code is not propagated and you may get an error message instead.
I suggest you use Execute() instead and check the exit code, not changing GetOutput()

Reply:
Done.